examples/make_import/README.md

```diff
@@ -24,8 +24,6 @@ Use the common help target:
         ENV       | VALUE |   DEFAULT   |                            SHORT DESCRIPTION                              
 ------------------+-------+-------------+---------------------------------------------------------------------------
   ARCH            |       | amd64       | Set the arch (Default from make)                                          
-  DEBIAN_FRONTEND |       |             | Set to noninteractive or stderr to null to enable non-interactive output  
   GO_VERSION      |       | 1.9.2       | Set the golang version (Default from make)                                
   K8S_VERSION     |       | v1.6.6      | Set the k8s version (Default from make)                                   
-  XDG_CACHE_HOME  |       | build/cache | Location to store/cache build assets                         
```

Comment:
Similar, this is a semi-standard way to specify a cache root, so is there a reason not to include this anymore?

Reply:
Once of the reasons was that this made the package more tied to a specific  implementation (freedesktop which is more of a linux thing).
If I revert the changes and move the environment variables back to the package globals this could be made a convenience again.

Reply:
TBH I'm not sure if there is a better variable to standardize on that is consistent with mac/linux, or whether we even want the cache used for builds to share in that cache directory. But I do suspect there is value to letting it be configurable by the user in a standard way.